plugins/inputs/jolokia/jolokia.go

```diff
@@ -41,20 +44,32 @@ func (c JolokiaClientImpl) MakeRequest(req *http.Request) (*http.Response, error
 type Jolokia struct {
 	jClient JolokiaClient
 	Context string
+	Mode    string
 	Servers []Server
 	Metrics []Metric
+	Proxy   Server
 }
 
 func (j *Jolokia) SampleConfig() string {
 	return `
-  ## This is the context root used to compose the jolokia url
-  context = "/jolokia/read"
+  # This is the context root used to compose the jolokia url
+  context = "/jolokia"
 
-  ## List of servers exposing jolokia read service
+  # This specifies the mode used
+  # mode = "proxy"
```

Comment:
Currently any value different to "proxy", make the input plugin executing in agent mode.